src/components/Header/Header.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+
+import './component.css'
```

Comment:
Even thought "Header" is a widely used component. Shouldn't it be a better option to give it its own css?

Reply:
It currently has its own .header class

If you mean css file, the component.css is a file that only contains the css related to header.

Reply:
We will standardize the component folder at a later date